resources/views/layout.blade.php

```diff
@@ -94,7 +94,7 @@
                     </li>
                 </ul>
 
-                @if(Auth::check() || Session::has('auth.vatsim-sso'))
+                @if(Auth::check() && Session::has('auth.vatsim-sso'))
```

Comment:
I changed it to `&&` because between SSO authentication and entering your password you are not authenticated but this check will still pass, meaning all calls to `Auth::user()` return null which breaks the page. Perhaps a separate dialogue box to log out if you are only authenticated through Vatsim SSO should be added?